.golangci.yml

```diff
@@ -28,4 +28,7 @@ issues:
   exclude-rules:
     - text: "Potential HTTP request made with variable url"
       linters:
-        - gosec
\ No newline at end of file
+        - gosec
+    - text: "type name will be used as volume.VolumeMount by other packages, and that stutters; consider calling this Mount"
```

Comment:
Let's remove this. I have a suggestion.

The original `Volume.go` does not really describe a docker volume as mentioned in `pkg/volume/volume.go`. 
 
I suggest renaming the file/struct as `DockerVolumeHelper` or something. Since it is nothing but a wrapper on calling helper methods to create/remove docker volume.

And once you do that. You can rename `volume_mount.go` to `volume.go` and `VolumeMount` to `Volume` which is more appropriate IMO.

What do you think?

Reply:
Sounds good to me. I'll refactor it.